src/apps/vpn/platforms/ios/ioscontroller.mm

```diff
@@ -93,10 +93,11 @@
             return;
         }
       }
-      callback:^(BOOL a_connected) {
+      callback:^(BOOL a_connected, NSDate* date) {
+        QDateTime qtDate(QDateTime::fromNSDate(date));
```

Comment:
What about if date is null. The doc does not cover this scenario. Can you please add an if-stmt and use the current date-time?

Reply:
I tested this initially, and just re-checked it. It works as expected - when that Swift optional returns nil, we get null all the way up. 

Then in `Controller::connected`, we hit this forking logic:
```
if (connectionTimestamp.isValid()) {
    m_connectedTimeInUTC = connectionTimestamp;
    emit timeChanged();
    m_timer.start(TIMER_MSEC);
  } else {
    resetConnectedTime();
  }
```
And without a valid timestamp, we get the time from `resetConnectedTime()` - which is where it was coming from (on initial iOS connection) before this PR as well.